MaskRCNN/config.py

```diff
@@ -133,6 +133,10 @@ def __ne__(self, _):
 _C.TRAIN.BATCH_SIZE_PER_GPU = 1
 _C.TRAIN.SEED = 1234
 _C.TRAIN.GRADIENT_CLIP = 0 # set non-zero value to enable gradient clip, 0.36 is recommended for 32x4
```

Comment:
Do we want TRAIN.GRADIENT_CLIP to be on or off by default?

Reply:
I think so. Customer (or someone else who use the repo) usually don't have too many nodes, it is not likely that they will hit the NaN problem.